ill-fated-ducks/workforce-management
# Mb dbinit

## Status
Choose:
READY


## Description
Db initializer
Class updates for consistency
Re-scaffolded to account for added ICollections 

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout ...
3. pull branch
4. build 
5. visit localhost:xxxxx/Computers then /Departments and so on to ensure it looks correct

## Impacted Areas in Application
Db Initializer, Controllers, Views (Automated)

@ill-fated-ducks
